scripts/data_prep/convert_delta_to_json.py

```diff
@@ -505,6 +508,12 @@ def fetch_DT(args: Namespace) -> None:
         help=
         'Use serverless or not. Make sure the workspace is entitled with serverless'
     )
+    parser.add_argument(
+        '--json_output_filename',
+        required=False,
+        type=str,
+        default='train-00000-of-00001.jsonl',
+        help='The combined final jsonl that combines all partitioned jsonl')
```

Comment:
@XiaohanZhangCMU Actually could you rename the `json_output_path` arg to be `json_output_folder` to be clear that the filename is a separate parameter? Or, you can combine and have `json_output_path='tmp/train-00000-of-00001.jsonl'` to be a simpler and better script.

Reply:
Done. I renamed it to json_output_folder to keep changes minimum. Please have a look.